### ISSUE TITLE
Hello, I want to ask a question about the semigan.

### ISSUE THREAD

**Issue by zhangqianhui** [sthalles/sthalles.github.io#1]
I have watched you introduction about semi-gan.
In my viewpoint,  A error has occured in your paper.

`gan_logits = tf.reduce_logsumexp(class_logits, 1) ...`

the value is not gan_logits. 

Do you think?

**Comment by sthalles** [comment]
Hello,

Yes, the result from the logsumexp is not the logits. The logits are the class_logits variable and they are computed just above this line. 

The logsumexp is the log likelihood and models the binary classification part of the GAN.

Luckily, I haven't mentioned that "gan_logits" are the real logits. Apparently, this is just a bad named variable. There is a much enhanced version of this article on Medium. Take a look if you wish: https://towardsdatascience.com/semi-supervised-learning-with-gans-9f3cb128c5e

Thanks for your input!

**Comment by zhangqianhui** [comment]
thanks
